Ошибка: HTTPConnectionPool(host='117', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x000002027AAE7550>: Failed to resolve '117' ([Errno 11001] getaddrinfo failed)"))